openassessment/data.py

```diff
@@ -993,15 +999,37 @@ def create_zip_with_attachments(cls, file, course_id, submission_files_data):
 
         with ZipFile(file, 'w') as zip_file:
             for file_data in submission_files_data:
```

Comment:
The `csvwriter` needs to be updated with the new header field. Right now it won't write `file_found` header so that information will be stripped from the CSV.

Reply:
I updated the `cls.DOWNLOADS_CSV_HEADER` variables in earlier commit.

Reply:
It's not showing in the diff, so I don't think the changes were committed correctly.

Reply:
LOL, JK. I'm still half asleep.